libs/barista-components/confirmation-dialog/src/confirmation-dialog.ts

```diff
@@ -177,6 +198,12 @@ export class DtConfirmationDialog
     this.state = null;
     this._overlayRef.dispose();
     this._stateChildrenSubscription.unsubscribe();
+    this._viewportChangesSubscription.unsubscribe();
+  }
+
+  /** Triggers a wiggle animation on the confirmation dialog */
+  wiggle(): void {
+    this._wiggleState = true;
   }
```

Comment:
After rethinking this during finalizing this PR - is a more generic function name like `focusAttention()` better - at some point we might decide to switch to another animation. Then wiggle is tightly coupled to the animation used.

Reply:
I've renamed the method to focusAttention and added the entry to the readme